todo_app/helpers/trello_items.py

```diff
@@ -0,0 +1,75 @@
+import os
+import requests
+from .item_class_helper import Card_Item
+
+api_key = os.getenv('TRELLO_API_KEY')
+api_token = os.getenv('TRELLO_API_TOKEN')
+board_id = os.getenv('BOARD_ID')
+list_id_to_do = os.getenv('TO_DO_LIST_ID') 
+list_id_done = os.getenv('DONE_LIST_ID')
+
+def get_items():
+    """
+    Fetches all saved items from the trello board.
+
+    Returns:
+        list: The list of saved items.
+    """
+    try:
+        response = requests.get(url = f"https://api.trello.com/1/boards/{board_id}/cards?key={api_key}&token={api_token}")
+        response.raise_for_status()
+        cards = response.json()
+        to_do_items=[]
+        for card in cards:
+            item = Card_Item.from_trello_card(card)
+            to_do_items.append(item)
+        return to_do_items
+    except Exception as e:
+        print(f"Getting items failed: {e}")
```

Comment:
Worth noting that if this gets triggered, the function returns None and the user gets a horrible error message anyway when the app tries to do something with that, so potentially re-raising the error after you've printed this message